skaha/client.py

```diff
@@ -30,43 +31,49 @@ class MyClient(SkahaClient):
 
     """
 
-    server = attrib(default="https://ws-uv.canfar.net/skaha")
-    certificate = attrib(
-        default="{HOME}/.ssl/cadcproxy.pem".format(HOME=environ["HOME"]), type=str
+    server: str = Field(default="https://ws-uv.canfar.net/skaha", title="Server URL")
```

Comment:
Rather than a string look into using `AnyHttpUrl` field type from pydantic, so that you can support both IP Addresses and HTTP Addresses.